.codesandbox/ci.json

```diff
@@ -1,6 +1,6 @@
 {
   "sandboxes": ["vanilla", "vanilla-ts"],
-  "node": "14",
+  "node": "18",
```

Comment:
This is probably outside the scope of this PR, but do we still need Codesandbox in the CI setup? It seems like unnecessary complexity.

Reply:
I have no idea, I would be happy to get rid of it for you if you in a separate PR if you decide it shouldn't stay.

Reply:
It gives us immediate releases to play around on every release, and people get sandboxes where those releases are already preinstalled.

Reply:
For this PR, that would be 
![grafik](https://github.com/reduxjs/react-redux/assets/4282439/05d269cd-4414-443c-a6d2-0a36f82c3ac0)

Unfortunately, it's a bit hidden:
![grafik](https://github.com/reduxjs/react-redux/assets/4282439/ae989d76-5900-40be-bbbd-0e6a1b8e7afc)